open_academy/views/partner_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="partner_view_form">
```

Comment:
As a good practice we use the same id than original record and add the suffix `_inherit` to indicates that is an inheritance. 

```suggestion
    <record model="ir.ui.view" id="view_partner_form_inherit">
```